ecl2df/common.py

```diff
@@ -66,6 +64,9 @@
     )
 
 
+# This is a magic filename that means read/write from/to stdout:
+__MAGIC_STDOUT__ = "-"
```

Comment:
Why the double dunder? Does this mean I cannot do this?

```python
from ecl2df.common import __MAGIC_STDOUT__

…
write_dframe_stdout_file(my_frame, __MAGIC_STDOUT__)
```

If that does not work, this is unfortunate.

Reply:
yes, that ddunder is probably misplaced.